### DISCUSSION TITLE
Vector sink with Loki gives 401 error

### DISCUSSION THREAD

**Discussion by gg2code** [vectordotdev/vector#21052] [Q&A]
I'm using vector v0.40.0, trying to connect to loki using the loki sink from openshift cluster, but getting 401 authentication error. I'm using basic auth.

**Comment by jszwedko** [comment]
A 401 would indicate that Loki didn't like the credentials 🤔

**Reply by gg2code** [reply]
I'm able to login through my terminal to loki, but get 401 error while connecting from vector sink.

**Reply by jszwedko** [reply]
I would set up a proxy like https://mitmproxy.org/ to examine the requests that Vector is making to see what the differences are.

**Reply by gg2code** [reply]
Issue with password, resolved now. Thanks.
